src/index.js

```diff
@@ -2,10 +2,12 @@ import { Controller } from 'stimulus'
 import formatDistanceToNow from 'date-fns/formatDistanceToNow'
 
 export default class extends Controller {
+  static isValid = true
```

Comment:
Why using a `static` here?  `initialize` sounds appropriate no?

```js
  initialize () {
    this.isValid = true
  }
```

Reply:
That's right. There is no need for `static` 👍🏼